soc/xtensa/espressif_esp32/esp32_net/Kconfig.series

```diff
@@ -8,5 +8,6 @@ config SOC_SERIES_ESP32_NET
 	select CLOCK_CONTROL
 	select DYNAMIC_INTERRUPTS
 	select HAS_ESPRESSIF_HAL
+	select NEED_POSIX_C_SOURCE if WIFI_ESP32
```

Comment:
@cfriedt, You can remove this from here and also from `esp_wifi_drv.c`.  No need for non-standard posix functions anymore in Espressif's sources. The entry in `esp_wifi_drv.c` was outdated I thinkl.

Reply:
Fixed!